VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
We've defined `*.opendb` earlier in the file - should that be more specific? Or is the general rule good enough?